config/job.go

```diff
@@ -73,13 +73,22 @@ type JobConfig struct {
 	WorkingDir string
 	// User Username or UID to use in the container. Format ``user[:group]``.
 	User string
+	//Devices  Maps the host devices you want to connect to a container
+	Devices []Device
```

Comment:
I think there should be a space before `Devices` here in the comment.

Could you add:

```
// type: list of device specs
// example: ``{Host: /dev/fb0, Container: /dev/fb0, Permissions: rwm}``
```

This text will be used to generate the reference docs